source/extensions/tracers/zipkin/span_buffer.cc

```diff
@@ -114,10 +114,13 @@ const std::vector<ProtobufWkt::Struct>
 JsonV2Serializer::toListOfSpans(const Span& zipkin_span, Util::Replacements& replacements) const {
```

Comment:
What is the story with having two serializers? I can't remember what the deprecation path looks like?

Reply:
I think the context was: https://github.com/envoyproxy/envoy/pull/6985/files#diff-14b9269528c04650b999a5344ba11168 before this, we only had v1 in mind.